modules/ROOT/pages/storage.adoc

```diff
@@ -128,3 +128,41 @@ storage:
 ----
 
 NOTE: You don't need the `path` or `with_mount_unit` keys; FCOS knows that the root partition is special and will figure out how to find it and mount it.
+
+== Encrypted storage (LUKS)
+
+Here is an example to configure a LUKS device at `/var/lib/data`.
+
+[source.yaml]
+----
+variant: fcos
+version: 1.2.0-experimental
+storage:
+  luks:
+    - name: data
+      device: /dev/vdb
+  filesystems:
+    - path: /var/lib/data
+      device: /dev/disk/by-id/dm-name-data
```

Comment:
Any reason for that vs `/dev/mapper/data`? I find the latter much nicer to use. :)

Reply:
Nope. I don't have a personal preference so switched them to `/dev/mapper/$device`.